src/app/public/modules/needs-attention/needs-attention.component.spec.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestModule
+} from '@skyux-sdk/builder/runtime/testing/browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  SkyNeedsAttentionComponent
+} from './needs-attention.component';
+
+describe('Needs attention component', () => {
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [SkyAppTestModule]
+    });
+  });
+
+  it('should use single column for one item', async () => {
```

Comment:
These don't need `async` unless you have a `fixture.whenStable` or another async action.